.travis.yml

```diff
@@ -2,7 +2,17 @@ language: generic
 
 script: bash ./deploy.sh
 
+branches:
+  only:
+    - gh-pages
```

Comment:
Maybe we can get away with *only* having `master`, since `deploy.sh` will push to `gh-pages`, which causes https://w3c.github.io/permissions/ to get the new document, and then echidna will pick it up from there.